src/lib/collab/provider.js

```diff
@@ -395,6 +395,7 @@ export class CollabProvider {
    */
   onLocalSave() {
     this.lastLocalSave = new Date()
+    console.log('onLocalSave', this.lastLocalSave)
```

Comment:
We should remove this log right?

Reply:
yep, this wasa mistake, see https://github.com/cozy/cozy-notes/pull/100